include/swift/Option/FrontendOptions.td

```diff
@@ -828,6 +828,11 @@ def enable_experimental_cxx_interop :
   HelpText<"Enable C++ interop code generation and config directives">,
   Flags<[FrontendOption, HelpHidden]>;
 
+def enable_cxx_interop :
+  Flag<["-"], "enable-cxx-interop">,
+  HelpText<"Alias for -enable-experimental-cxx-interop">,
+  Flags<[FrontendOption, HelpHidden]>;
```

Comment:
It is really important that we have the word "experimental" in this flag. This is an expirmental feature, we need to make sure that's communicated to people using it. 

Slightly related: I also want to add a warning anytime someone enables this.